stl.go

```diff
@@ -210,6 +216,9 @@ func ReadFromSTL(i io.Reader) (o *Subtitles, err error) {
 		STLTimecodeStartOfProgramme:                         g.timecodeStartOfProgramme,
 		Title:                                               g.originalProgramTitle,
 	}
+	if opts.IgnoreTimecodeStartOfProgramme {
```

Comment:
I'd rather remove line 216 and instead add

```go
if !opts.IgnoreTimecodeStartOfProgramme {
  o.Metadata.STLTimecodeStartOfProgramme = g.timecodeStartOfProgramme
}
```

Reply:
OK. Makes sense. I've changed that.